### DISCUSSION TITLE
How to add ssl configure to support https

### DISCUSSION THREAD

**Discussion by yuetaibai** [vuejs/vitepress#1398] [Q&A] [answered]
Thanks in advance~

I need to add the ssl certificate to support https access from internet, here is ./docs/.vitepress/config.js

```js
module.exports = {
  server: {
      https: {
        cert: fs.readFileSync(path.join(__dirname, 'cert/cert.crt')),
        key: fs.readFileSync(path.join(__dirname, 'cert/cert.key')),
        ca: fs.readFileSync(path.join(__dirname, 'cert/ca.crt')),
      }
    }
}
```

here is ./package.json
```js
{
  "scripts": {
    "docs:dev": "vitepress dev docs --port 3002 --https",
  },
}
```
With the above config, run the server by "yarn docs:dev --host" successfully. But when access the Vitepress, always prompt "ERR_SSL_VERSION_OR_CIPHER_MISMATCH"

Any help would be appreciated !

**Comment by brc-dd** [comment, marked as answer]
refer https://github.com/vuejs/vitepress/issues/1403#issuecomment-1258315979
